### DISCUSSION TITLE
How to be sure that I’m using the right decoder

### DISCUSSION THREAD

**Discussion by ElPiix** [LizardByte/community#95] [Sunshine] [answered]
Hi, 
First thanks a lot for your work! The game streaming works great, I'm using a RX 6650 XT and was just wondering if I was using the right encoder. Let me explain: in the settings I forced the AMD AMF/VCE, but on my Moonlight client it remains displayed OMX.MTK.VIDEO.DECODER.HEVC. Is this normal? Sorry if my question is stupid but I'm a little new with all this... Thank you in advance for your answers!

**Comment by ReenigneArcher** [comment, marked as answer]
Encoder is on the host, decoder is on the client. You can check the stdout (output) of the `sunshine` command to determine what encoder is being used.

**Comment by ElPiix** [comment]
> Encoder is on the host, decoder is on the client. You can check the stdout (output) of the `sunshine` command to determine what encoder is being used.

Great thanks for the information! 👌🏻
